Stratis.Bitcoin.Features.RPC.Tests/Controller/GetStakingInfoActionTests.cs

```diff
@@ -48,16 +45,13 @@ public void GetStakingInfo_StakingEnabled()
         /// Tests that the RPC controller of a staking node correctly replies to "startstaking" command.
         /// </summary>
         [Fact]
-        public void GetStakingInfo_StartStaking()
+        public async Task GetStakingInfo_StartStakingAsync()
         {
             string dir = AssureEmptyDir("TestData/GetStakingInfoActionTests/GetStakingInfo_StartStaking");
             IFullNode fullNode = this.BuildStakingNode(dir, false);
             var node = fullNode as FullNode;
 
-            Task.Run(() =>
-            {
-                fullNode.Run();
-            });
+            await fullNode.RunAsync();
```

Comment:
This is probably not correct as this method should not block.
Did the test pass?

Reply:
Fixed at: https://github.com/stratisproject/StratisBitcoinFullNode/pull/581